### PR TITLE
add respawn method to MouseClick script

### PR DESCRIPTION
**Pull request by Kintaro-Oe**

### CODE REVIEW

File: Guacamole/Assets/Scripts/MouseClick.cs [hunk 1]
```diff
@@ -29,6 +31,12 @@ void OnMouseDown()
 
     void CreateMole()
     {
+        Respawn();
```
Comment (GuiHeurich) [inline Guacamole/Assets/Scripts/MouseClick.cs:34]: Great method name.

File: Guacamole/Assets/Scripts/MouseClick.cs [hunk 2]
```diff
@@ -29,6 +31,12 @@ void OnMouseDown()
...
         mole.gameObject.SetActive(true);
     }
+
+    public void Respawn()
+    {
+        this.transform.position = spawnPoint.position;
+    }
```
Comment (GuiHeurich) [inline Guacamole/Assets/Scripts/MouseClick.cs:41]: Is this "this" referring to the mole?

**Review by GuiHeurich — APPROVED** [review]
Nice work.

File: Guacamole/Assets/Scripts/MouseClick.cs (hunk 2 above)
Comment (Kintaro-Oe) [reply Guacamole/Assets/Scripts/MouseClick.cs:41]: In this case yes!
"This" refers to whichever object that the script is attached to.

**Pull request merged**